### ISSUE TITLE
Pattern: Content Group Card - Accessibility - The heading and the body in each card is given as link.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#1382]
## Detailed description
Device - JAWS FF, VO iPhone Safari

Issue :- The heading and the content in each card is given as link. This makes it redundant. 
Suggestion:-  
1. If both the links in each card is expected to perform the same action, there is no need to provide different links. Either the heading or the content under the heading can be given as link. 
2. It is not recommendable to given the entire textual content as a link. Because it makes the screen reader over verbose. The link text or the label should be concise. 

## Additional information
 - The solution for this should be the same as issue (#1482)

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/3b7800c0-1fc8-4fbc-a452-224bf2e7748d)

**Comment by RobertaJHahn** [comment]
@oliviaflory Can you verify if this is a problem from an accessibility standpoint that has to be fixed.

**Comment by oliviaflory** [comment]
@RobertaJHahn I tried to do some research on the accessibility site about repetitive links, but am having difficulty finding a recommendation. 

cc @wonilsuhibm

**Comment by wonilsuhibm** [comment]
This is a bad UX for users using screen readers at the minimum. This should be read once as a whole.

**Comment by RobertaJHahn** [comment]
Assigned a sev 4. Can be fixed in the future release.

**Comment by philljenkins** [comment]
Also consider the mobile experience - are there two spots for taping on mobile with blank spots in-between that could incorrectly be attempted to be tapped with no results?  
Why can't the whole card be the clickable / tapable region? Then the screen reader only hears the heading (link label) once.

**Comment by ljcarot** [comment]
Yes, we want the whole card to be clickable but only one link label so we have to investigate how to fix it.

**Comment by RaphaelAmadeu** [comment]
@Praveen-sr 

Could you please test this again?
We added an aria-label on the link,

Thanks!

**Comment by Praveen-sr** [comment]
@RaphaelAmadeu Ram re tested this on [experimental](https://ibmdotcom-react-experimental.mybluemix.net/?path=/story/patterns-blocks-contentgroupcards--default) and [staging](https://ibmdotcom-react-staging.mybluemix.net/?path=/story/patterns-blocks-contentgroupcards--default) but Screen reader is not identifying the link in each card, all are announced only as text.

**Comment by RaphaelAmadeu** [comment]
@Praveen-sr 
Here's the link,
https://ibmdotcom-react.s3.us-south.cloud-object-storage.appdomain.cloud/deploy-previews/1877/index.html

Thanks!

**Comment by RaphaelAmadeu** [comment]
@Praveen-sr 
Do we have any updates on this?
Thanks!

**Comment by Praveen-sr** [comment]
@RaphaelAmadeu Yes, ram verified on the link you provided, but Screen reader is not identifying the link in each card, all are announced only as text.

**Comment by RaphaelAmadeu** [comment]
@Praveen-sr 
Another update was made,
Could you please test this once more?
Thanks!

**Comment by Praveen-sr** [comment]
@RaphaelAmadeu Ram verified this, issue still exists. Can we have a discussion on call and resolve it?
Thanks.

**Comment by RaphaelAmadeu** [comment]
@RobertaJHahn 
After investigating this with @kennylam we found out that this is an issue inherited from Carbon,
They are documented working on the `Card` used component `Tile`
It will be fixed, as soon as they update the component,

The same can be said for the issue #1482
Thank you!
